crates/sui-types/src/gas.rs

```diff
@@ -97,18 +105,26 @@ struct SuiCostTable {
     /// TODO: We should introduce a flat fee on storage that does not get refunded even
     /// when objects are deleted. This cost covers the cost of storing transaction metadata
     /// which will always be there even after the objects are deleted.
-    pub storage_per_byte_cost: StorageCost,
+    pub storage_per_byte_non_refundable_cost: StorageCost,
+    /// This is the refundable piece of the storage cost
+    pub _storage_per_byte_refundable_cost: StorageCost,
```

Comment:
Curious how do we plan to use this. Would we introduce another field in GasCostSummary as well in the result?

Reply:
Yes